CustomDataTable.py

```diff
@@ -0,0 +1,354 @@
+#####
+#NOTES: PETER BUSCHE
+    # 1) Deleted all comments/documentation from previous author. 
+        # Reason: Previous comments/documentation are preserved in boilerplate code. If we update and funcionalities, 
+                #  we don't want leftover comments to potentially confuse us. Also, starting here but subject to change,
+                # we know that all commented out code is the old code that I updated that was throwing errors. Thereofore,
+                # we can quickly see what changed without having to use a comparison tool to the old code. For now, we can put 
+                # comments up here.
+
+    # 2) repr() vs str()? 
+        # I would prefer is everything was str() 
+        # if we update all repr() to str() will this cause errors later?
+
+    # 3) All changes that involve the use of super()
+        # I am not sure if this provides the correct functionality. I am trusting ChatGPT with this change.
+        # Examples:
+            # super(CustomDataTable, self).__init__() 
+            # super(Genealogy, self).__init__(data, log, header)
+            # super(CustTableGrid, self).__init__(parent, pos=pos, size=size)
+
+
+
+
+# ERRORS:
+    # 1) line 154: call to KinGroups
+        # when we fix KinGroups and allow import statement this should be resolved
+
+    # 2) line 211: call to pyp_newclasses
+        # when we fix pyp_newclasses and allow import statement this should be resolved
+####
+
+
+
+import wx.grid
+from wx.grid import *
+import os
+from relate import KinGroups
```

Comment:
I'm not sure if Python is case strict or not, but the file in the project is `Relate.py`